debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/oracle/generated/PlSqlParser.g4

```diff
@@ -2157,7 +2157,40 @@ access_driver_type
 
 external_table_data_props
     : DEFAULT DIRECTORY quoted_string
-    | LOCATION
+    | ACCESS PARAMETERS '(' record_format_spec? field_format_spec? ')'
+    | LOCATION '(' quoted_string ')'
+    ;
```

Comment:
While this syntax works, I question whether it is exactly correct.  At least according to [the documentation](https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/CREATE-TABLE.html#GUID-F9CE0CC3-13AE-4744-A43C-EAC7A71AAAB6), the `external_table_data_props` rule can technically be a combination of all 3 of these rules.  So I wonder if this shouldn't be written differently to account for that?

Also, we should probably consider adding the `USING CLOB subquery` rule as a part of the `ACCESS PARAMETRS` path that goes alongside the record and field format specs.

Also can you share where you found the syntax for `LOCATION`, it doesn't seem to align with 19c.

Reply:
> So we adhere to 19c documentation for Oracle DDL parser?

We always need to check both 12c and 19c for consistency but we should always adhere to the latest syntax and add special handling where necessary for older versions that will eventually be obsolete imo. 

That documentation link is for Oracle 9i which we do not support.  The best way to find the syntax is to always google for something like `CREATE TABLE syntax Oracle 19c`.  You can also always go directly to the [Oracle 19c documentation](https://docs.oracle.com/en/database/oracle/oracle-database/19/index.html) and search for `CREATE TABLE` if that's much easier.  

For the `opaque_format_spec`, you can use [this landing page](https://docs.oracle.com/en/database/oracle/oracle-database/19/sutil/oracle-external-tables-concepts.html#GUID-07D30CE6-128D-426F-8B76-B13E1C53BD5A) as a reference.  The formats are dictated by the `ORACLE_LOADER`, `ORACLE_DATAPUMP`, and `ORACLE_HDFS` or `ORACLE_HIVE` drivers.  You'll see on this page there is a link to each driver and each gives a description of its parameters.  These parameters can be pretty extensive to support so do your best.  I would say if anything we should at least have the loader flushed out given that it's pretty common to use SQL loader with Oracle to do bulk data loads.  You'll find on the see-also link for that one there is a complete diagram description of the various clause specs.

Lastly regarding `SK` vs `SKIP`, I don't see anything anywhere in any of the docs that indicate that `SK` is valid.  That doesn't mean that this isn't some silly nuance by Oracle LogMiner with its re-construction of the SQL.  You could allow both `SKIP` and `SK` or reach out to the reporter to confirm; however, I'm pretty sure Raffael would have provided it from the logs as it was written.

Reply:
Thanks, @Naros. I will update the syntax accordingly.

> however, I'm pretty sure Raffael would have provided it from the logs as it was written.

To that, I had to first fix some of the typos, like missing `,` and spelling errors for `varchar`. So I wasn't sure about `SK` being a type error or actually something. I will reach out to the reporter for the confirmation of the same.